XIVComboExpanded/Combos/WHM.cs

```diff
@@ -188,7 +191,8 @@ internal class WhiteMageGlare4Feature : CustomCombo
 
     protected override uint Invoke(uint actionID, uint lastComboMove, float comboTime, byte level)
     {
-        if (actionID == WHM.Glare3)
+        if (actionID == WHM.Stone || actionID == WHM.Stone2 || actionID == WHM.Stone3 ||
```

Comment:
I usually had Stone only, and had decided to only put Glare3 there because you're not supposed to have any lower skills anyway when you're leveled up enough to get Glare 4 ready lmao but at least that's consistent now

Reply:
...I'll be honest, I definitely thought that was the Dia block (clearly was coding late), but ya, it doesn't really harm anything =P